README.md

```diff
@@ -1,17 +1,47 @@
 
 # An introduction to RAG with Elastic
-### Presentation: [Presentation_name](workshop/Presentation_template.pptx)
+### Presentation: [Introduction to RAG](workshop/Introduction_to_RAG.pdf)
 
 ## Workshop description
 During the workshop you will learn what is Retrieval Augmented Generation, how it can increase trustability of LLM models and how to set up a RAG pipeline using Elastic.
 
 ## Requirements
-Do not forget to indicate Python version and any other tools
-+ add requirements.txt or conda.yml or docker image or Binder/Google Collab link
+
+Python version: >=3.8.5
+
```

Comment:
Please add _requirements.txt_ or _conda.yaml_ or Binder/Google Collab link. Also, some users may not have Jupyter installed to start the notebook.

Reply:
done